ext/storage_sources/azure_store/azure_connection.h

```diff
@@ -0,0 +1,24 @@
+#ifndef AZURECONNECTION
```

Comment:
Let's remove this, since I don't think we need it and its not part of C++ guidelines. We do however need to add a #pragma once.

Reply:
+1

Reply:
Resolved :)